internal/db/schema/migrations/oss/postgres/79/02_update_time_indexes.up.sql

```diff
@@ -0,0 +1,39 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+create index if not exists auth_ldap_account_update_time_ix on auth_ldap_account (update_time);
```

Comment:
We use the  `_idx` suffix for the indices created in the other migration file. I see there is precedent for either in our SQL migrations, but what do you think about usinx `_idx` here for consistency within this migration group?